bundles/org.openhab.binding.emby/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="emby"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<!-- Sample Thing Type -->
+	<bridge-type id="controller">
+		<label>EMBY Server</label>
+		<description>This is the Bridge to an instance of an EMBY server you want to connect to.</description>
+		<config-description>
+			<parameter name="api" type="text" required="true">
+				<label>API Key</label>
+				<description>This is the API key generated from EMBY used for Authorization.</description>
+			</parameter>
+			<parameter name="bufferSize" type="integer" min="1024">
+				<label>WebSocket Buffer Size</label>
+				<description>Size of the WebSocket buffer in bytes.</description>
+				<default>10000</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>Server Host</label>
+				<description>IP address or hostname of the EMBY server.</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65535" required="true">
+				<label>Server Port</label>
+				<description>Port number for the EMBY server.</description>
+				<default>8096</default>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" min="1000">
+				<label>Refresh Interval</label>
+				<description>Polling interval for play-state updates.</description>
+				<default>10000</default>
+			</parameter>
+			<parameter name="discovery" type="boolean">
+				<label>Auto Discover</label>
+				<description>Enable or disable automatic device discovery.</description>
+				<default>true</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="device">
```

Comment:
Can you add equipment tags to both things?
For reference: https://next.openhab.org/docs/developer/bindings/thing-xml.html#thing-semantic-equipment-tags

Reply:
done